test/main.test.js

```diff
@@ -250,6 +251,10 @@ describe("sqlparser", function () {
         } // Leave t4 "as is"
       );
       deepStrictEqual(tables, ["t1", "t2", "t3", "t4"]);
+      deepStrictEqual(statements, [
+        'select `t1`.id,[t2].* from `t1` join [t2] join "t3" join(select * from t4)'
+      ]);
+      strictEqual(type, "read");
```

Comment:
These two assertions are not currently on main or staging.  I had added them as a way to double check the return values of `normalize`, happy to set this up as a separate PR if that's cleaner.